ios/Classes/Blemulator/Model/Characteristic.h

```diff
@@ -0,0 +1,20 @@
+#import <CoreBluetooth/CoreBluetooth.h>
+#import "Service.h"
+
+@interface Characteristic : NSObject
```

Comment:
Was it not possible to reuse the class from MBA?

Reply:
It's a part of RxBluetoothKit and it's not exposed to objc (would require some changes in RxBluetoothKit)